pysetup/spec_builders/electra.py

```diff
@@ -10,3 +11,13 @@ def imports(cls, preset_name: str):
         return f'''
 from eth2spec.deneb import {preset_name} as deneb
 '''
+
+## TODO: deal with changed gindices
+
+    @classmethod
+    def hardcoded_ssz_dep_constants(cls) -> Dict[str, str]:
+        return {
+            'FINALIZED_ROOT_GINDEX': 'GeneralizedIndex(169)',
+            'CURRENT_SYNC_COMMITTEE_GINDEX': 'GeneralizedIndex(86)',
+            'NEXT_SYNC_COMMITTEE_GINDEX': 'GeneralizedIndex(87)',
```

Comment:
/cc @etan-status 

The gindex will be updated due to the increased number of fields. We should update it to `FINALIZED_ROOT_GINDEX_ELECTRA`. 

Would this be a problem for the light client?

Reply:
Not a problem, the light client protocol is designed to be forward compatible, as in, the existing `v1` REST endpoints and libp2p endpoints continue to work fine, even if these indices adjust with new forks. However, the sync-protocol.md needs to take the new constant into account though, for both proof generation as well as verification and in the data structures.

Further keep in mind that, due to the periodic nature of these reindexings, without EIP-7495, it is impossible to create an immutable verifier that lives on a different chain / hardware wallet / low-cadence software (EIP-7671 etc), that can consume proofs without needing to be continuously maintained. As we are aiming to break indices with Electra, I'd suggest to prioritize EIP-7495 for Electra as well, and make `BeaconState`, `BeaconBlockBody` etc `StableContainer` so that no further index changes are expected as long as the fundamental architecture stays the same.

This way, we will re-index these containers once (with Electra), but it is the final time that we have to deal with this busywork that just introduces churn at arbitrary times without adding value. We shouldn't have to ever worry about breaking some unrelated component just because a new feature gets added or an old feature gets removed.

On EIP-7495, would appreciate reviews. If we want to transition `BeaconState` etc to `StableContainer`, probably makes sense to push the `Variant` type beyond just a notation but also give it more compact serialization. as in, if we know that an Electra variant is sent, no need for all the leading bits everywhere.